stylesheets/components/alerts/alerts.css

```diff
@@ -0,0 +1,92 @@
+@import "/stylesheets/index.css";
+
+.general-alerts{
+    margin-bottom: 0;
+}
+
+.simple-alerts {
+    background: var(--simple-alerts-section);
+    padding: 1rem;
+    border-radius: 1rem;
+}
+
+.alerts-simple {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0px;
```

Comment:
do we need to give gap 0px? isn't it the default setting?

Reply:
It is. I have a common class in addition that takes 2rem which I'd need for the other components too. Only here I'd need no gap, hence 0px.